Make.inc

```diff
@@ -49,6 +49,13 @@ CFLAGS_add += -std=c99 -Wall -I$(OPENLIBM_HOME) -I$(OPENLIBM_HOME)/include -I$(O
 
 default: all
 
+# *int / *intf need to be built with -O0
```

Comment:
Yes. See the failing test for `nearbyint` below that I had commented out a while back, and passes as a result of this.